src/components/create-guild/hooks/useCreateGuild.tsx

```diff
@@ -20,7 +20,7 @@ const useCreateGuild = () => {
   const fetcherWithSign = useFetcherWithSign()
 
   const fetchData = async (signedValidation: SignedValdation): Promise<Guild> =>
-    fetcher("/guild", signedValidation)
+    fetcher("/v2/guilds/with-roles", signedValidation)
```

Comment:
Oh, one more note here: I think we'll need to update the `cypress/e2e/0-platformless/0-create-guild.spec.ts` test file too, it is failing rn ;)